dashboard/src/types/tree/TestDetails.tsx

```diff
@@ -3,16 +3,17 @@ export type TTestDetails = {
   build_id: string;
   compiler: string;
   config_name: string;
-  environment_misc: string | undefined;
   git_commit_hash: string;
   git_repository_branch: string;
   git_repository_url: string;
   id: string;
   log_excerpt: string | undefined;
   log_url: string | undefined;
-  misc: string | undefined;
   path: string;
   start_time: string;
   status: string;
   environment_compatible?: string[];
+  environment_misc?: object;
+  misc?: object;
+  output_files?: object;
```

Comment:
`Record<string, unknown>` is more accurate than object for this

https://www.typescriptlang.org/play/?target=7#code/PTAEE0HsFdQYwIYDtQIM5oJYHMXIJ4AuAFpktqIZKAAYDeAvjQDSgBUApgB5wcAOhNqABmkAE60k0ADbSaqJABNa0JR2FkOimgDoAsACg4kJGkKoAXKACCYsQnwAeRgD5QAXlABtQ6D+gAIjMxMmwA5l9-QjFoDgiDf1AAFgAmeMTUpHT-AGV8AFsAI0hpAAoASmy-LwBGVhSAXSrvJsi-OlFIKwDChDEAhmbGZoqPN2HDNtApWWbVRXVNRXiGgG5JgxAIGHhkVAwcPCQiUnJKYgRCAHI0diRIcwRQPhD8zEJMADcOISpaSEKACsOHBCLpDFsAAqvd5fDi3PocUDBUJoVjFEocZBo6bQIocMQ4wo4UBkQgcbAEnFoArFaQ4mgzOQKZQ0eaLJBacFGExmUCFKy2exOAHA0FuTw+BL+Wr1VrS6ryxIdSBdQK9fqDKYTBWgUbucZa3VTILRULhKbRWLNVI2lJZKZ5IolCrNJlzNQaTnLQxrDZbKCwRAoEzSfD7LC4diikGENC-ag0ABKIPEikcKPIrFUAGt7gB3JAubnGUzmOCCuwORwp4xidOZ7DZpB5yCFlwS7zazrdDUDIZGxImxsW3VWuJTW2T+3NJ1011Td1TdlerTNWWgRrrpX+fWGlarIA